docs/source/tools_integration/ipython.md

```diff
@@ -97,8 +97,8 @@ Out[3]:
  'example_learning_rate': 0.01}
 ```
 
-```eval_rst
-.. tip:: If you enable `versioning <../data/data_catalog.md#versioning-datasets-and-ml-models>`_ then you can load a particular version of a dataset, e.g. :code:`catalog.load("example_train_x", version="2021-12-13T15.08.09.255Z")`.
+```{note}
```

Comment:
From documentation contributor guidelines: "Prefer to use `note`, `warning` and `important` only, rather than a number of different colours/types of callout." I tried to color match to the appropriate level.